packages/account/src/Components/forms/form-fields/date-of-birth-field.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Field, FieldInputProps, FormikHelpers, FormikState } from 'formik';
+import { DateOfBirthPicker, TDatePickerOnChangeEvent } from '@deriv/components';
+import { toMoment } from '@deriv/shared';
+
+type TDateOfBirthFieldProps = {
+    name: string;
+    portal_id: string;
+} & Omit<React.ComponentProps<typeof DateOfBirthPicker>, 'onBlur' | 'onChange' | 'error'>;
+
+type TDateOfBirthFieldHelpers = {
+    field: FieldInputProps<string | moment.Moment>;
+    form: FormikHelpers<{ date_of_birth: string | moment.Moment }> &
+        FormikState<{ date_of_birth: string | moment.Moment }>;
+};
+
+/**
+ * DateOfBirthField is a wrapper around DateOfBirthPicker that can be used with Formik.
+ * @name DateOfBirthField
+ * @param name - Name of the field
+ * @param portal_id - Portal ID
```

Comment:
have we agreed to add doc for types and component description?

Reply:
This gives better intellisence when a developer uses the component.